src/main/java/net/dv8tion/jda/internal/requests/restaction/MessageActionImpl.java

```diff
@@ -280,6 +293,48 @@ public MessageActionImpl override(final boolean bool)
         return this;
     }
 
+    @Nonnull
+    @Override
+    public MessageAction allowedMentions(@Nullable EnumSet<Message.MentionType> allowedMentions)
+    {
+        this.allowedMentions = allowedMentions == null ? null : allowedMentions.clone();
```

Comment:
When you change this to accept collections you can switch to `EnumSet.copyOf(collection)`